lab-dustin/test/hero-router-test.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+require('dotenv').config({path: `${__dirname}/../.test.env`});
+//npm modules
+const expect = require('expect');
+const superagent = require('superagent');
+// app modules
+const Hero = require('../model/hero.js');
+const server = require('../lib/server.js');
+
+let tempHero;
+const API_URL = process.env.API_URL;
+
+describe('testing hero router', () => {
+  before(server.start);
+  after(server.stop);
+
+  describe('testing GET /api/heros/:id', () => {
+    var tempHero;
+
+    afterEach(() => Hero.remove({}));
+    beforeEach(() => {
+      return new Hero({
+        name: 'The Juicer',
+        origin: 'The Kitchen',
+        goodOrEvil: 'Evil',
+      })
+      .save()
+      .then(hero => {
+        tempHero = hero;
+      });
```

Comment:
With one resource, this is viable. When we start getting into bigger projects with more resources this stuff should be in it's own setup file.